### ISSUE TITLE
INCOMPATIBILIDADE react-native-svg E react-native-svg-charts

### ISSUE THREAD

**Issue by WilliamHeleodoro** [JesperLekland/react-native-svg-charts#608]
Olá.
Estou com um problema de incompatibilidade nas versões. Quando dou um npm install ele perde uma versão do react-native-svg: 6.2.1||^7.0.3
Porém se coloco nessa versão ele da erro no carregamento do Circle e outros componentes do react-native-svg devido a utilizar proptypes, acredito que esteja sendo descontinuado as proptypes.

Se eu volto para a versão 13.9.0 do react-native-svg da certo, porém quando vou gerar uma apk pelo expo, da erro nas dependências.
Consegue me ajudar como posso resolver isso?

**Comment by rohith-thummanapally** [comment]
hey, i am also gettting  similar issue , were you able to resolve it, if so , can you explain how have you resolved it

**Comment by Ohmniox** [comment]
Yes, I am also getting the same error. I need the latest version of react-native-svg (^14.1.0) and when I try to install this package, it fails with error:
```
Could not resolve dependency:
npm ERR! peer react-native-svg@"^6.2.1||^7.0.3" from react-native-svg-charts@5.4.0
```

**Comment by Wolk9** [comment]
Somone should change the dependency to: 
```
{
   "react-native-svg": ">=13.9.0"
   .....
}

```
as Expo needs this version at least, and this repo doesn't seem to break with this version of react-native-svg

**Comment by zhenjianlee** [comment]
I am having the same issue. Does anyone have a solution to this?

**Comment by mpaulbrown21** [comment]
Also having this issue

**Comment by salahbm** [comment]
Same here, having this issue with react-native-svg latest version...

**Comment by salahbm** [comment]
However, `npm install --save react-native-svg-charts --legacy-peer-deps` does a trick with some npm errors....

**Comment by gaboAcosta** [comment]
So, this package is abandoned correct?

**Comment by luiztafarel** [comment]
alguem conseguiu alguma solução?
